.github/workflows/pr.yml

```diff
@@ -1,9 +1,15 @@
-name: PR Build
+name: "Build and Test"
 on:
   pull_request:
     branches:
       - master
       - staging
+  push:
+    branches:
+      - master
+      - staging
+env:
+  CROSS_BUILD: ${{ github.ref == 'refs/heads/staging' || github.ref == 'refs/heads/master' || startsWith(github.head_ref, 'preview') }}
```

Comment:
Right now, you don't use this env. Maybe we can use it in `if` statement at `crossCompile` job ?

Reply:
We use it in ciRunSbt.sh - all tests && publishing will be done with "+" then